README.md

```diff
@@ -19,11 +19,11 @@ Yep! Macmoji substituions are ordinary text substituions. Double click an entry
 
 Some fun recommendations:
 
-| replace | with |
-| :------- | :------- |
-| ehh | ¯\_(ツ)_/¯ |
-| tflip | (╯°□°）╯︵ ┻━┻ |
-| eml | your@email.com |
+| replace  | with           |
+| :------- | :------------- |
+| ehh      | ¯\\\_(ツ)_/¯   |
```

Comment:
the second one doesn't technically need to be escaped since markdown will leave unmatched underscores alone, but it may be preferable anyway for consistency's sake.